lib/Autoloader.php

```diff
@@ -58,6 +63,51 @@ public static function removePackage(Package $package)
 	}
 
 	// }}}
+	// {{{ public static function addFinalPackage()
+
+	/**
+	 * Sets an autoloader package as the last package to check against
+	 *
+	 * @param Package $package the autoloader package to add.
+	 *
+	 * @return void
+	 */
+	public static function setFinalPackage(Package $package)
+	{
+		self::$final_package = $package;
+	}
+
+	// }}}
+	// {{{ public static function removeFinalPackage()
+
+	/**
+	 * Removes the final autoloader package from the autoloader
+	 *
+	 * @return void
+	 */
+	public static function removeFinalPackage()
```

Comment:
This method can be removed if `setFinalPackage` is updated to modify `self::$packages`